wallet-new/src/Cardano/Wallet/API/Request/Filter.hs

```diff
@@ -145,10 +146,12 @@ instance Show (FilterOperation ix a) where
     show (FilterByRange _ _)          = "FilterByRange"
     show                              = formatToString build
 
+deriveSafeBuildableExt $ \v -> [t| FilterOperation $v $v |]
```

Comment:
Looks much better right before `BuildableSafeGen` instance :heart:

Reply:
This looks like it'll make an instance like:

```haskell
instance SafeBuildable (FilterOperation t t)
```

which won't ever be satisfied :\

Reply:
`v` is not a pure value here (surprisingly), but is `newName "x" :: Q Type` (approximately).
I believe that those `$v` will result in different types.